### ISSUE TITLE
file URI scheme

### ISSUE THREAD

**Issue by tsmgeek** [ezyang/htmlpurifier#145]
I am trying to put file:// in an <img> tag but it keeps clearing it out.
I am using purifier to generate XML documents with valid html for a description, the img tags reference locally downloaded files as file://imgage.jpg
What is wrong here?

**Comment by ezyang** [comment]
file:// URLs are not supported by default. Take a look at URIScheme to see what goes into defining something.

**Comment by tsmgeek** [comment]
Currently I have to do this to enable file urls.
```
$schemeRegistry = HTMLPurifier_URISchemeRegistry::instance();
$schemeRegistry->getScheme('file',$config,null)->browsable=true;
```

Can we just have a config param URI.AllowFileLinks?

**Comment by ezyang** [comment]
Yes, I suppose so. Better name for the param would be more like URI.LocalFileBrowsable, default false. I'd accept a patch.
